main.nf

```diff
@@ -17,12 +18,12 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/mhcquant --mzmls '*.mzML' --fasta '*.fasta' --vcf '*.vcf' --class_1_alleles 'alleles.tsv' --include_proteins_from_vcf --predict_class_1 --refine_fdr_on_predicted_subset -profile standard,docker
+    nextflow run nf-core/mhcquant --sample_sheet 'sample_sheet.tsv' --fasta 'SWISSPROT_2020.fasta'  --allele_sheet 'alleles.tsv'  --predict_class_1  --refine_fdr_on_predicted_subset -profile standard,docker
 
     Mandatory arguments:
-      --mzmls [file]                            Path to input data (must be surrounded with quotes)	
-      --raw_input [bool]                        Specify whether raw files should be used as input
-      --raw_files [file]                        Path to input raw data (must be surrounded with quotes)
+      --sample_sheet [file]                     Path to sample sheet (must be surrounded with quotes)
+      --raw_input [bool]                        Specify whether raw files are used as input
```

Comment:
Would it make sense to merge raw_input and mzml_input to a single 'spectra' or as 'input' (longer discussion) and just identify the file format (raw or mzml) on the fly and use the appropriate entry point?

Reply:
yep that is what I also wanted to ask

Reply:
ok yep I remember the discussion, I'll have another look at the proteomicsLFQ

Reply:
ok we changed it to input